arenadata/README.md

```diff
@@ -12,5 +12,6 @@ To run the tests, execute the script:
 <path_to_pgbackrest>/arenadata/run_docker.sh <image_id> [arch type]
 ```
 [arch type] stands for CPU architecture. Default value is x86-64. This argument is needed only for log files namings.
+[image_id] supports both GPDB 6 (`gpdb6_u22:latest`) and GPDB 7 (`gpdb7_u22:latest`).
```

Comment:
Could you please additionally change <image_id> to image name, since docker image id (hash) does not suit here.
And could you please change the default image name in `run_docker.sh` from gpdb6_regress to ubuntu version naming for 6x. 

And, ofc, describe the changes in patch description